app/routers/categories.py

```diff
@@ -32,7 +34,7 @@ class Config:
 
 
 # TODO(issue#29): get current user_id from session
-@router.get("/")
+@router.get("/categories/by_parameters")
```

Comment:
I wonder.. if there are 2 get requests with different params (this one and `@router.get("/categories")`, 
can't they have the same route?